xray_vision/mpl_plotting.py

```diff
@@ -120,9 +123,130 @@ def show_label_array(ax, label_array, cmap=None, **kwargs):
     vmin = max(.5, kwargs.pop('vmin', .5))
 
     ax.set_aspect('equal')
-    im = ax.imshow(label_array, cmap=cmap,
+    im = ax.imshow(label_array, cmap=_cmap,
                    interpolation='nearest',
                    vmin=vmin,
                    **kwargs)
 
     return im
+
+
+def binary_state_lines(ax, chrom_data, xmin=0, xmax=120,
+                       delta_y=3,
+                       off_color="#1C2F4D",
+                       on_color="#FA9B00",
+                       lc_kwargs=None):
```

Comment:
The downside of `None` as a default is that the user can't tell what the defaults actually are without looking at the source code.